### ISSUE TITLE
MultiInput Component: underlying HTML changes needed for a11y

### ISSUE THREAD

**Issue by jbadan** [SAP/fundamental#1353]
MultiInput component: https://sap.github.io/fundamental/components/multi-input.html

`<span role='button'>` should be `<button>` element for the tokens:

```
    <div class="fd-multi-input-tags">
        <span class="fd-token" role="button">Apple</span>
        <span class="fd-token" role="button">Berry</span>
        <span class="fd-token" role="button">Banana</span>
    </div>
```

This will be solved by #1346, but the examples inside of MultiInput should be updated to reflect this.
